src/test/java/uk/gov/hmcts/reform/bulkscan/utils/TestDataUtil.java

```diff
@@ -858,4 +858,66 @@ public static List<OcrDataField> getA58Data() {
 
         return fieldList;
     }
+
+    public static List<OcrDataField> getA58PostPlacementData() {
+        List<OcrDataField> fieldList = new ArrayList<>();
+        OcrDataField ocrDataFirstNameField = new OcrDataField();
+        ocrDataFirstNameField.setName("applicant1_firstName");
+        ocrDataFirstNameField.setValue("firstName");
+        fieldList.add(ocrDataFirstNameField);
+
+        OcrDataField ocrDataLastNameField = new OcrDataField();
+        ocrDataLastNameField.setName("applicant1_lastName");
+        ocrDataLastNameField.setValue("LastName");
+        fieldList.add(ocrDataLastNameField);
+
+        OcrDataField ocrContactNumberField = new OcrDataField();
+        ocrContactNumberField.setName("applicant1_telephoneNumber");
+        ocrContactNumberField.setValue("+447405878672");
+        fieldList.add(ocrContactNumberField);
+
+        OcrDataField ocrAddressField = new OcrDataField();
+        ocrAddressField.setName("applicant1_address");
+        ocrAddressField.setValue("123 test street, London");
+        fieldList.add(ocrAddressField);
+
+        OcrDataField birthMotherFirstNameField = new OcrDataField();
+        birthMotherFirstNameField.setName("mother_firstName");
+        birthMotherFirstNameField.setValue("testBirthMotherFirstName");
+        fieldList.add(birthMotherFirstNameField);
+
+        OcrDataField birthMotherLastNameField = new OcrDataField();
+        birthMotherLastNameField.setName("mother_lastName");
+        birthMotherLastNameField.setValue("testBirthMotherLastName");
+        fieldList.add(birthMotherLastNameField);
+
+        OcrDataField birthMotherAddressField = new OcrDataField();
+        birthMotherAddressField.setName("mother_address");
+        birthMotherAddressField.setValue("testBirthMotherAddress");
+        fieldList.add(birthMotherAddressField);
+
+        OcrDataField birthMotherNationalityField = new OcrDataField();
+        birthMotherNationalityField.setName("mother_nationality");
+        birthMotherNationalityField.setValue("testBirthMotherNationality");
+        fieldList.add(birthMotherNationalityField);
+
+        OcrDataField birthMotherOccupationField = new OcrDataField();
+        birthMotherOccupationField.setName("mother_occupation");
+        birthMotherOccupationField.setValue("testBirthMotherOccupation");
+        fieldList.add(birthMotherOccupationField);
+
+        OcrDataField childRelationshipFatherField = new OcrDataField();
+        childRelationshipFatherField.setName("child_relationship_father");
+        childRelationshipFatherField.setValue("true");
+        fieldList.add(childRelationshipFatherField);
+
+        return fieldList;
```

Comment:
can we use request json for per test case ?

Reply:
done